crates/exec-wasmtime/src/config.rs

```diff
@@ -6,10 +6,14 @@ use std::{collections::HashMap, ops::Deref};
 use serde::{de::Error as _, Deserialize, Deserializer};
 use url::Url;
 
-fn default_port() -> u16 {
+const fn default_port() -> u16 {
```

Comment:
I think we should just remove it and instead use `default_value` on L107 https://github.com/enarx/enarx/pull/2008/files#diff-1dffd47a20f4acb91d39396f05a61ec4b063f041570242e918b3b8ebb5631428R107